server/src/main/java/com/epam/aidial/core/server/controller/ControllerSelector.java

```diff
@@ -74,7 +74,7 @@ public class ControllerSelector {
 
     private static final Pattern USER_INFO = Pattern.compile("^/v1/user/info$");
 
-    private static final Pattern APP_SCHEMAS = Pattern.compile("^/v1/custom_application_schemas(/schemas|/meta_schema)?$");
+    private static final Pattern APP_SCHEMAS = Pattern.compile("^/v1/application_type_schemas(/schemas|/schema|/meta_schema)?$");
```

Comment:
please make group after /. That will improve tracing.

Reply:
done